### PR TITLE
Print addons path as absolute path instead of relative

### PR DESCRIPTION
**Pull request by florian-dacosta**
@bealdav @sebastienbeau @hparfr @bguillot 

Because it is so annoying it does not work that way...
I did not really understand the logic with './' in spec.yaml.
I did try to keep the previous logic, even if I think it is outdated. And I believe it should be removed
Anyway, I let these comments in the code source.

### CODE REVIEW

**Comment by florian-dacosta** [comment]
/* delete force push log

**Comment by bguillot** [comment]
Why those force pushs ?

**Comment by florian-dacosta** [comment]
/* delete force push log

**Comment by sebastienbeau** [comment]
@hparfr  what do you think?

**Comment by hparfr** [comment]
I don't remember the goal of this PR.

./something_different_that_external-src  is good feature. It let you manage other things like custom python libs and stuff.

By the way, the print was mean to be just an helper for easing the transition to the links.
I'm more in favor for removing the print feature instead.

**Comment by florian-dacosta** [comment]
Hi, I don't remember exactly the goal.
I think it was to have a correct addons_path with absolute path. Because relative path in odoo config is an issue, as you will have to be in odoo/ to lauch the server.
With absolute path, you can lauch odoo server from anywhere.

Regarding removing the feature because the transition to the links is done... I disagree.
1) All projects are not migrated yet.
2) the links stuff is good for project in production, but not necessarily for development purpose.
For developments (new project, or just developping some oca modules not linked to any project) I don't use the links feature.

**Comment by hparfr** [comment]
> For developments (new project, or just developping some oca modules not linked to any project) I don't use the links feature.
But do you use the print links feature ?

**Comment by florian-dacosta** [comment]
Well, it it gave the right addons_path, I would use it!
But not with the relative addons_path

File: ak/ak_build.py [hunk 1]
```diff
@@ -135,21 +136,30 @@ def _print_addons_path(self, config):
...
 
-        addons_path = ','.join(paths)
-        print('Addons path for your config file: ', addons_path)
-        return addons_path
+        # Construct absolute path, then relative path if any and merge both
+        # AFAIK the relative path logic should be removed.
+        # Because we do not use it and it add complexity
+        addons_path = '{}{}'.format(PREFIX, ',{}'.format(PREFIX).join(paths))
+        relative_path = ','.join(relative_paths)
+        whole_path = '{},{}'.format(addons_path, relative_path)
```
Comment (hparfr) [inline ak/ak_build.py:160]: You can achieve like that: 
```py

print_abs_path = ",".join(['/odoo/' + path for path in paths])
print('Addons path for your config file: ', print_abs_path) 
```
In that way, it's less complex and don't change the return value of this method.

Comment (florian-dacosta) [reply ak/ak_build.py]: @hparfr 
What about now?
Notice, I don't print the relative path at all. As you said, if this feature is used, it is more likely for custom lib or stuff, but probably not odoo module, so it should not be in the addons_path.
Or we could make a second print if we think it is useful, personally, I don't think it is, but I don't mind printing stuff!

Also, I now return the same stuff as before, but I am not sure it is very consistent.
First the return value is never used, and ak is not so used outside akretion and even less overridden, so I don't think it is an issue to change this return value right now, but also, seems strange to print absolute path and return something else.
Actually, returning a value in a method called "print" is kind of strange also, maybe we should just remove the return value.

Comment (hparfr) [reply ak/ak_build.py]: >Also, I now return the same stuff as before, but I am not sure it is very consistent.
First the return value is never used, and ak is not so used outside akretion and even less overridden, so I don't think it is an issue to change this return value right now, but also, seems strange to print absolute path and return something else.
Actually, returning a value in a method called "print" is kind of strange also, maybe we should just remove the return value.
Correct.

**Pull request merged**